src/utils/definitions/general_config.rs

```diff
@@ -0,0 +1,92 @@
+use cairo_felt::Felt252;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use tokio::sync::OnceCell;
+
+use super::constants;
+
+const _GENERAL_CONFIG_FILE_NAME: &str = "general_config.yml";
+const _N_STEPS_RESOURCE: &str = "n_steps";
+// const DEFAULT_CHAIN_ID: Felt252 = Felt252::new(0); // Fix this
+
+// Default configuration values.
+
+pub const DEFAULT_VALIDATE_MAX_STEPS: usize = 10usize.pow(6);
+pub const DEFAULT_TX_MAX_STEPS: usize = 3 * 10usize.pow(6);
+pub const DEFAULT_ENFORCE_L1_FEE: bool = true;
+
+// Given in units of wei
+pub const DEFAULT_GAS_PRICE: usize = 10usize.pow(8);
+
+#[derive(Debug, Serialize, Deserialize)]
+struct StarknetOsConfig {
+    chain_id: Felt252,
+    fee_token_address: Felt252,
+}
+
+// TODO: add real values
+impl Default for StarknetOsConfig {
+    fn default() -> Self {
+        StarknetOsConfig {
+            chain_id: Felt252::new(0),
+            fee_token_address: Felt252::new(0),
+        }
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct StarknetGeneralConfig {
+    starknet_os_config: StarknetOsConfig,
+    contract_storage_commitment_tree_height: usize,
```

Comment:
avoid using usize

Reply:
why ?

Reply:
it's better to have a fixed size on every machine